IdentityCore/src/logger/MSIDLogger+Internal.h

```diff
@@ -76,6 +79,15 @@ MSID_LOG(MSIDLogLevelVerbose, _correlationId, nil, NO, NO, _fmt, ##__VA_ARGS__)
 #define MSID_LOG_VERBOSE_PII(_ctx, _fmt, ...) \
 MSID_LOG(MSIDLogLevelVerbose, nil, _ctx, YES, NO, _fmt, ##__VA_ARGS__)
 
+
+// New macroses
+#define MSID_LOG_WITH_CONTEXT(_LVL, _CONTEXT, _PII, _FMT, ...) [[MSIDLogger sharedLogger] logWithLevel:_LVL context:_CONTEXT correlationId:nil containsPII:_PII format:_FMT, ##__VA_ARGS__]
+#define MSID_LOG_WITH_CORRELATION(_LVL, _CORRELATION_ID, _PII, _FMT, ...) [[MSIDLogger sharedLogger] logWithLevel:_LVL context:nil correlationId:_CORRELATION_ID containsPII:_PII format:_FMT, ##__VA_ARGS__]
```

Comment:
after this, we are removing all of the other macros before, correct?

Reply:
yes :)